src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +29,15 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn- str-to-int [s]
+  #?(:clj (Integer/parseInt s)
+     :cljs (js/parseInt s 10)))
+
+(defn parse-sequence-number-within-year-to-number [external-id]
+  (str-to-int (get (str/split external-id #"/") 1)))
+
+;; put test here
+(deftest test-validate-external-id
```

Comment:
The unit test should always be named `test-the-function-name` so here `test-parse-sequence-number-within-year-to-number`.

Reply:
ok!